.babelrc

```diff
@@ -7,8 +7,10 @@
         "targets": {
           "node": "4.3"
         },
+        "include": [
+          "transform-async-to-generator"
```

Comment:
This change is only needed for the test to use `async/await` ? Just as a note for now `webpack-defaults` will override this every time it runs